demo-project/conf/prod/globals.yml

```diff
@@ -1 +1 @@
-base_location: s3://my_bucket/production/
\ No newline at end of file
+_base_location: s3://my_bucket/production/
```

Comment:
This file should be `catalog_globals.yml`

Reply:
Good catch :+1: ! This file was renamed accordingly in the last commit.